ci/gpu/build.sh

```diff
@@ -86,8 +86,8 @@ gpuci_mamba_retry install -y \
                   "ucx-py=0.23.*"
 
 # https://docs.rapids.ai/maintainers/depmgmt/
-# gpuci_mamba_retry remove --force rapids-build-env rapids-notebook-env
-# gpuci_mamba_retry install -y "your-pkg=1.0.0"
+gpuci_mamba_retry remove --force rapids-build-env rapids-notebook-env
```

Comment:
```suggestion
gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
```

This needs to be conda, instead of mamba ( https://github.com/rapidsai/ops/issues/1918 )